Src/library/cURL/interface/curl_externals_i.e

```diff
@@ -253,9 +354,158 @@ feature {NONE} -- C externals
+				#endif
+			]"
+		end
+
+	c_mime_type (a_api: POINTER; a_curl_handle: POINTER; a_mimetype: POINTER): INTEGER
+			--	CURLcode curl_mime_type(curl_mimepart * part , const char * mimetype );
+		require
+			a_api_exists: not a_api.is_default_pointer
+			exists: not a_curl_handle.is_default_pointer
+		external
+			"C inline use <curl/curl.h>"
+		alias
+			"[
+				#ifdef CURL_STATICLIB
+					return (FUNCTION_CAST(CURLcode, (curl_mimepart *, const char *)) $a_api)((curl_mimepart *)$a_curl_handle, (const char *) $a_mimetype);
+				#else
+						/* Using proper calling convention for dynamic module */
+					return (FUNCTION_CAST(CURLcode, (curl_mimepart *, const char *)) $a_api)((curl_mimepart *)$a_curl_handle, (const char *) $a_mimetype);
+				#endif
+			]"
+		end
+
+	c_mime_subparts (a_api: POINTER; a_curl_handle: POINTER; a_subparts: POINTER): INTEGER
+			--	CURLcode curl_mime_subparts(curl_mimepart * part , curl_mime * subparts );
+		require
+			a_api_exists: not a_api.is_default_pointer
+			exists_mimepart: not a_curl_handle.is_default_pointer
+			exists_subparts: not a_subparts.is_default_pointer
+		external
+			"C inline use <curl/curl.h>"
+		alias
+			"[
+				#ifdef CURL_STATICLIB
+					return (FUNCTION_CAST(CURLcode, (curl_mimepart *, curl_mime * )) $a_api)((curl_mimepart *)$a_curl_handle, (curl_mime *) $a_subparts);
+				#else
+						/* Using proper calling convention for dynamic module */
+					return (FUNCTION_CAST(CURLcode, (curl_mimepart *, curl_mime *)) $a_api)((curl_mimepart *)$a_curl_handle, (curl_mime *) $a_subparts);
+				#endif
+			]"
+		end
+
+
+	c_mime_headers (a_api: POINTER; a_curl_handle: POINTER; a_headers: POINTER; a_take_ownership: INTEGER): INTEGER
+			-- CURLcode curl_mime_headers(curl_mimepart * part , struct curl_slist * headers , int take_ownership );
+		require
+			a_api_exists: not a_api.is_default_pointer
+			exists_mimepart: not a_curl_handle.is_default_pointer
+			exists_headers: not a_headers.is_default_pointer
+		external
+			"C inline use <curl/curl.h>"
+		alias
+			"[
+				#ifdef CURL_STATICLIB
+					return (FUNCTION_CAST(CURLcode, (curl_mimepart *, struct curl_slist * , int )) $a_api)((curl_mimepart *)$a_curl_handle, (struct curl_slist *) $a_headers, $a_take_ownership);
+				#else
+						/* Using proper calling convention for dynamic module */
+					return (FUNCTION_CAST(CURLcode, (curl_mimepart *, struct curl_slist * , int)) $a_api)((curl_mimepart *)$a_curl_handle, (struct curl_slist *) $a_headers, $a_take_ownership);
+				#endif
+			]"
+		end
+
+	c_mime_filedata (a_api: POINTER; a_curl_handle: POINTER; a_filename: POINTER): INTEGER
+			-- CURLcode curl_mime_filedata(curl_mimepart * part , const char * filename );
+		require
+			a_api_exists: not a_api.is_default_pointer
+			exists_mimepart: not a_curl_handle.is_default_pointer
+		external
+				"C inline use <curl/curl.h>"
+			alias
```

Comment:
bad indentation for the whole alias block

Reply:
Fixed